README.md

```diff
@@ -104,6 +104,42 @@ Here is a complete example:
 </body>
 ```
 
+<br/>
+However, in larger applications, to ensure the iframe and the build method are called in the same JavaScript event loop, you can create the iframe and set its src, then call build and capture its promise, before finally appending the iframe and awaiting the captured promise.
```

Comment:
This is a hefty sentence. I would maybe simplify this for a basic user. Especially if this is the first page/thing they are going to be reader for the package. Here is a very rough idea of maybe something that I would considered more "simplified" below. 

However, in larger applications you maybe want to ensure the iframe and the PublisherInterface build method are called in the same JavaScript event loop. To do this you can
1. Create the iframe element and set the src attribute for the iframe
2. Pass the iframe element to the build method of PublisherInterface and capture the promise returned from the method
3. Appending the iframe element to the DOM
4. Await the promise captured in step 2 and assign it to a variable to be used throughout your application